lib/config/options/index.ts

```diff
@@ -382,6 +382,7 @@ const options: RenovateOptions[] = [
     globalOnly: true,
     type: 'string',
     default: null,
+    stage: 'global',
```

Comment:
This makes sure that random `logContext` values don't stay in the config and cachebust